src/components/SquarePaymentForm.test.tsx

```diff
@@ -88,8 +88,8 @@ describe('SquarePaymentForm', () => {
   })
 
   describe('methodsSupported', () => {
-    ['applePay', 'googlePay', 'masterpass'].forEach(method => {
-      ['ready', 'unavailable'].forEach(state => {
+    ;['applePay', 'googlePay', 'masterpass'].forEach(method => {
```

Comment:
This feels weird to me. is there a reason we need the semicolons on the start of each line? I haven't seen that convention before.

Reply:
I thought it was a bug too but it looks like it's intentional:

https://github.com/prettier/prettier/issues/4316

Reply:
😮